### ISSUE TITLE
Cross-Platform Multi-Currency Wallet and Transfer Bot for Chimoney

### ISSUE THREAD

**Issue by amosayomide05** [Chimoney/chimoney-community-projects#394]
### What would you like to share?

Create a bot that supports Telegram, WhatsApp, and Discord, allowing users to seamlessly send and receive multi-currency funds across these platforms. Using their given Telegram, WhatsApp, or Discord IDs as unique identifiers (like account numbers), users can transfer money between these platforms. The bot will handle user authentication, balance inquiries, and notifications across these platforms. The bot utilize Chimoney’s API for secure transfers and wallet management, ensuring encrypted transactions between platforms.

This is an upgrade for #290 and #197

@phyleria kindly assign me to this issue

**Comment by phyleria** [comment]
Hi @amosayomide05 

Sure, go ahead. I've assigned it to you.

**Comment by amosayomide05** [comment]
> Hi @amosayomide05 
> 
> Sure, go ahead. I've assigned it to you.

Ok thanks

**Comment by phyleria** [comment]
Yes, I'll still be reviewing PRs past Hacktoberfest. So go ahead please.
